transforms/helpers/schema-helper.ts

```diff
@@ -0,0 +1,40 @@
+import { inspect } from 'node:util';
+import type { ZodEffects, ZodTypeAny } from 'zod';
+import { z } from 'zod';
+
+const preprocessStringToBoolean = <I extends ZodTypeAny>(
```

Comment:
This file seems like a good place to DRY up some of the union error message duplication.

Reply:
Correct. I tried it but didn't like the built-in error messages.